changelog/12251.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+identity: allow creating a role with a non-existent key
```

Comment:
This was always the case, up until #12151 was introduced right? Since this is a bug fix for a PR within 1.8.1 and results in no behavioral change from the user, I wonder if we can just `pr/no-changelog` this one.

Reply:
Correct, thanks